packages/flutter_tools/test/base/build_test.dart

```diff
@@ -190,7 +190,6 @@ void main() {
       expect(genSnapshot.callCount, 1);
       expect(genSnapshot.snapshotType.platform, TargetPlatform.ios);
       expect(genSnapshot.snapshotType.mode, BuildMode.profile);
-      expect(genSnapshot.packagesPath, '.packages');
```

Comment:
Can the field in `genSnapshot` also be deleted (and other related code)?

Reply:
See also comments here https://github.com/flutter/flutter/pull/28261#issuecomment-466081968

Reply:
Just did a commit for removing the packagePath related stuff from _FakeGenSnapshot.

Digging through the actual build code to see if there's other places that the packageMap could be removed in `build.dart`, or if there are non snapshot-generating parts of the build that still need to have this information.

At a quick glance, it's unclear to me whether the uses of `packagePath` in, e.g., `AOTSnapshotter.build()` are still needed to make sure that the correct paths are generated for the `sky_engine` package, etc. for use by the `Fingerprinter()` and others. That is, if a different `packagesPath` than `.packages` was used by a previous compilation before the snapshot is generated, is that path stored in the results in a way that it's picked up there by the `gen_snapshot` process and thus we can't assume `.packages` when doing the fingerprint checks (and other checks) mentioned above?

(If that's not a concern and basically `.packages` is a forced place for packages now, then I can change it appropriately to just assume that.)